### ISSUE TITLE
Refactoring of the moving3D skill

### ISSUE THREAD

**Issue by ptaillandier** [gama-platform/gama#2581]
**Is your request related to a problem? Please describe.**
the code of the Moving3D skill should be improved (lot of unnecessary lines of code).

**Comment by agrignard** [comment]
Yes it can be shorter indeed. Don't hesitate to modify it. Regarding this specific model I also notice that the Goto on the 3D network seems to be broken. Agent used to be able to go from origin to destination using the 3D graph but they don't do that anymore (I don't exactly what is going on , it seems that they go directly to the XY plan and do not stay on there initial Z location).

**Comment by AlexisDrogoul** [comment]
Tracked under #2206.

**Comment by AlexisDrogoul** [comment]
Reopening this issue for the next round of development on GAMA 2.0
